recipes-bsp/u-boot/u-boot-hab.inc

```diff
@@ -23,9 +23,9 @@ UBOOT_BINARY_HAB = "${UBOOT_BINARY}"
 UBOOT_BINARY_HAB:mx6dl-generic-bsp = "u-boot-ivt.img"
 UBOOT_BINARY_HAB:mx6q-generic-bsp = "u-boot-ivt.img"
 
-# Sign U-Boot binary and/or SPL.
+# Sign U-Boot binary and/or SPL (this works both with i.MX6 and i.MX7).
 # This is supposed to run in the U-Boot build directory.
-sign_habv4() {
+imx6_sign_habv4() {
```

Comment:
Since this also applies to iMX7, makes sense o rename it to something like `imx6_imx7_sign_habv4`?

Reply:
Absolutely! I'll do that.

Reply:
Updated as per suggestion.